client/src/Components/UserList/actions.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+
+export const LOADING = "LOADING";
+export const FOUND_USERS = "FOUND_USERS";
+export const ERROR = "ERROR";
+
+export const fetchUsers = () => {
+  return dispatch => {
+    dispatch({
+      type: LOADING
+    });
+    axios
+      .get("http://138.197.202.158/api/users/all")
```

Comment:
Following our Restful convention, all of the endpoints (except for user registration and personal user endpoints) will be `URL/api/resource-names`

Reply:
changed path to /api/users